{
    "id": "bundle--06c9d619-42c3-40da-9594-625bd69b7f36",
    "objects": [
        {
            "created": "2021-03-13T20:11:51.3887Z",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "description": "An issue was discovered in GitLab Community Edition (CE) and Enterprise Edition (EE) 12.3 through 12.6.1. It allows Denial of Service.",
            "external_references": [
                {
                    "external_id": "CVE-2019-20142",
                    "source_name": "cve"
                }
            ],
            "id": "vulnerability--e559d2f4-d24d-4eaa-84ed-2ff4073957fe",
            "modified": "2021-11-22T14:54:52.900149Z",
            "name": "CVE-2019-20142",
            "spec_version": "2.1",
            "type": "vulnerability"
        }
    ],
    "type": "bundle"
}